debian/changelog

```diff
@@ -1,3 +1,26 @@
+mir (0.31.2) UNRELEASED; urgency=medium
+
+  * New upstream release 0.31.1(https://github.com/orgs/MirServer/projects/4)
```

Comment:
this says 0.31.1 and its the link is 404 (maybe its a private project?)

Reply:
Whoops, I just copied it. Should I remove that line or change it to something? What does it even mean?

Reply:
Remove the link, we don't have projects for bugfix releases: https://github.com/MirServer/mir/projects